### DISCUSSION TITLE
How to listen to localhost only?

### DISCUSSION THREAD

**Discussion by HubKing** [usememos/memos#2360] [Q&A] [answered]
The default conf file has only MEMOS_PORT. How can I make this localhost only? I am going to use NginX to enable SSL.

**Comment by lincolnthalles** [comment, marked as answer]
`MEMOS_ADDR="127.0.0.1"`

**Comment by sp823** [comment]
I'm sure I'm doing something totally wrong but I am trying to do the same thing and I tried adding MEMOS_ADDR='127.0.0.1' as an environmental variable to the docker run commandline and it is just totally unreachable when I do that, including from the local machine, so the reverse proxy fails. Where is MEMOS_ADDR supposed to go?

**Reply by HubKing** [reply]
I installed it directly without Docker, so in my case the config file is at `/etc/memos.conf`.

**Comment by sp823** [comment]
Thanks for the tip - I got a list of files in the docker container hoping I could maybe edit it and unfortunately I don't have any memos.conf file in there.
